### PR TITLE
Login scenario improvement

### PR DESCRIPTION
**Pull request by GreyYar**
Fixed Login feature
Fixed lint warning

### CODE REVIEW

File: features/page/homePage.js [hunk 1]
```diff
@@ -64,7 +64,7 @@ const homePage = Object.create({
       .elementIdElements(parentElementId, amountOfMessagesNumberSelector);
     const lastNumberOfMessageOnPage = Number(headerMessageOnMessages.value[1].getText());
     const firstNumberOfMessageOnPage = Number(headerMessageOnMessages.value[0].getText());
-    return lastNumberOfMessageOnPage - (firstNumberOfMessageOnPage + 1);
+    return lastNumberOfMessageOnPage - (firstNumberOfMessageOnPage - 1);
```
Comment (AleksandrHorev) [inline features/page/homePage.js]: why?

File: features/login.feature [hunk 2]
```diff
@@ -1,4 +1,4 @@
-
+@watch
```
Comment (AleksandrHorev) [inline features/login.feature]: it's needed?

File: features/page/homePage.js (hunk 1 above)
Comment (GreyYar) [reply features/page/homePage.js]: Was working incorrectly (first number was 1, last is 100, 100 - (1 + 1) = 98 was compared with 100 mails on the page which failed tests)

File: features/login.feature (hunk 2 above)
Comment (GreyYar) [reply features/login.feature]: Now it works as expected and could be included

**Pull request merged**